Fix hang in apiv2 test_connect

cevich opened the pull request:
When executing on F36, starting the podman service in debug-mode causes aardvark to run in debug mode.  This does unexpected things with file-descriptors leading to a test-hang.  Thanks to @Luap99 for the fix.  See also https://github.com/containers/automation_images/pull/128 and https://github.com/containers/podman/pull/13934

Fixes https://github.com/containers/podman/issues/13932

jwhonce commented:
LGTM

TomSweeneyRedHat commented:
@cevich, rebase needed.

Luap99 commented on Makefile:
```diff
@@ -616,18 +616,24 @@ remotesystem:
-	# Order is important running python tests first causes the bash tests to fail, see 12-imagesMore
-	# FIXME order of tests should not matter
+.PHONY: localapiv2-1
+localapiv2-1:
```
Can we name this more descriptive, i.e. `localapiv2-bash` and `localapiv2-python` instead

cevich replied:
Good idea, sure thing.

cevich commented:
This should be good to go.

cevich commented:
Force-pushed: Fixed commits and removed timeouts for @TomSweeneyRedHat per https://github.com/containers/podman/pull/13376#pullrequestreview-949312922

TomSweeneyRedHat commented:
/approve

TomSweeneyRedHat commented:
LGTM
and happy green test buttons!

Luap99 commented:
/lgtm
